src/encoded/static/components/dataset.js

```diff
@@ -2868,12 +2870,15 @@ export const SeriesComponent = ({
                     supplementalShortLabels={supplementalShortLabels}
                     showReplicateNumber={false}
                     hideControls={!METADATA_SERIES_TYPES.includes(context['@type'][0])}
-                    collapseNone
```

Comment:
This is funny, I guess this isn't used anywhere. Maybe it's left from some previous change that did need it.

Reply:
The rules about collapsing have changed in this ticket, so that’s why this went away.